caregiver_application/README.md

```diff
@@ -11,4 +11,5 @@
 	`cd caregiver_application`
 - Run the created .jar file in the target folder:  
 	`python main.py`  
-- Access the app at: http://localhost:5000.
\ No newline at end of file
+- Access the app at: http://localhost:5000.
+- Use testmodels folder to upload the BPMN and DMN files  for this version
```

Comment:
This should not be the case. All models must be on a central place and all need to work. the name of the model should indicate what it contains